proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -1,10 +1,10 @@
 syntax = "proto3";
-package cosmos.tx;
+package cosmos.tx.v1beta1;
```

Comment:
What is the reason for this not being in `cosmos.base.tx.v1beta1` ? It seems to me like this would be most inline with other conventions (e.g. `cosmos.base.query.v1beta1` for generic querying / pagination related definitions).

Reply:
See https://github.com/cosmos/cosmos-sdk/pull/6905#issuecomment-668085344